base_decorator.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require_relative 'nameable'
+
```

Comment:
- [ ] Add  `attr_accessor :nameable` to your code so that namemble can be getter and setter